osm2gtfs/core/osm_connector.py

```diff
@@ -42,7 +42,11 @@ def __init__(self, config):
         # tags from config file for querying
         self.tags = ''
         for key, value in self.config["query"].get("tags", {}).iteritems():
-            self.tags += unicode('["' + key + '" = "' + value + '"]')
+            if isinstance(value, list):
+                value = '|'.join(value)
+                self.tags += unicode('["' + key + '" ~ "' + value + '"]')
```

Comment:
Why do you switch to `~` ?

It would work with a `=` here too, and would be safer : we don't want Overpass to match regular expression and get matches that partially fit the values.

Reply:
> ...and get matches that partially fit the values.

Well, I've simply followed the suggestion by @jamescr in [#110](https://github.com/grote/osm2gtfs/issues/110#issuecomment-353234482), but you are right, that's indeed a problem.

I'll look into it.

Reply:
I've changed it accordingly and added regex anchors (`^` and `$`) to prevent partially matching.

Reply:
Thanks a lot for noticing it!